### ISSUE TITLE
SAMM for product builders

### ISSUE THREAD

**Issue by maximbaele** [owaspsamm/samm-suite#63]
**Description:**
SAMM 2.x is focusing on organizations that have and-to-end contol over the software they create and operate.
These organizations own the full development lifecycle of their products, from inception to operation, such as webapp builders. In the meantime, we see the model being used for all types of organizations and teams, including those whose products are operated by customers. I'll call them "product builders" for now. 

I'd like to see the next version of SAMM to treat these "product builders" as first-class citizens, not requiring additional guidance outside of the core model, not requiring extensive interpretation from practitioners to make the model "fit". This will also facilitate SAMM being used in the context of product security legislation such as EU CRA.

The way I use the model with products builders is as follows;

_Every product has a lifecycle that SAMM aims to cover end-to-end, but the team in scope will only be directly responsible for part of that lifecycle. That does not mean the product’s lifecycle stops when the product is transferred to the customer, it is just someone else’s responsibility to securely deploy, configure, operate, monitor, update and decommission the product. Whether the customer succeeds in doing that entirely depends on the provisions the developing team provides in their part of the development lifecycle. 

This means that teams need to consider how their products will be used, and how that would be done in a secure way. How does the team ensure that the customer can verify the authenticity of the product? What types of (security) logs and integrations are provided? Do customers get the right documentation to securely configure and operate the product? Do customers receive vulnerability alerts and patches?_  

**Acceptance criteria:**
The SAMM core model does not require guidance outside of the core model to be useful for product builders, nor does it require extensive interpretation of the practices and quality criteria.

**Are there any known dependencies between this issue and any other issues?**
Not that I am aware of, but it is 100% a breaking change.

**Are there any outstanding questions?**
/

**Comment by aganariman** [comment]
Deployment and operations of products quite often are taken care by independent departments in big corporates and indeed it's difficult to do assessment of both development and operations in a single scope. Would be nice to update operations activities to make them clear for teams that focus only on building software products.

**Comment by ottosulin** [comment]
I agree that it could be useful for many companies building software-enabled physical products, such as many IoT type devices, to have a bit clearer examples and references to their work. Current content is applicable as such already, but can require a bit more interpretation and experience from practitioners - we could make this easier and approachable to new practitioners.

I'm certain that once EU CRA and EU Product Liability Directive are drawing closer to enforcement in the following years, there will be a lot more interested in SAMM because there will be real financial penalties for shipping faulty software, not providing security updates etc.
